gen/solid.hoon

```diff
@@ -11,26 +11,44 @@
   !:
 :-  %say
 |=  $:  {now/@da eny/@uvJ bec/beak}
-        {$~ $~}
+        {arg/$@($~ {top/path $~}) dub/_|}
```

Comment:
This could use a doccord explaining that `top` is the top-level arvo directory, defaulting to `/=home=/sys`. `dub` is a flag that will cause `+solid` twice to catch bugs in the hoon compiler.